examples/flutter_gallery/lib/demo/expansion_panels_demo.dart

```diff
@@ -191,17 +191,15 @@ class _ExpansionPanelsDemoState extends State<ExpasionPanelsDemo> {
 
           return new CollapsibleBody(
             margin: const EdgeInsets.symmetric(horizontal: 16.0),
-            child: new Input(
-              hintText: item.hint,
-              labelText: item.name,
-              value: new InputValue(
-                text: item.value
+            child: new Form(
+              child: new Input(
+                hintText: item.hint,
+                labelText: item.name,
+                value: new InputValue(text: item.value),
```

Comment:
I don't think this is correct.  Creating a new InputValue each time will lose information about the selection and editing state.  Is there a need to use a Form here?  Why not just store the InputValue as state and return the InputValue you get in the onChanged callback?